README.md

```diff
@@ -26,21 +26,32 @@ Use [rvm](https://rvm.io/) to install the version of Ruby specified in `.ruby-ve
 
  1. Copy `config/flickr.yml.example` to `config/flickr.yml` and update the fields with your Flickr credentials.
 
-### Gems
+### Docker
 
-We use bundler to manage gems. You can install bundler and other required gems like this:
+Docker can be used to: 1) run just the required services (MySQL, Elasticsearch, etc.) while [running the asis application in your local machine](https://github.com/GSA/asis#developmentusage), and/or 2) run the entire `asis` application in a Docker container.  Please refer to [searchgov-services](https://github.com/GSA/search-services) for detailed instructions on centralized configuration for the services.
 
-    gem install bundler
-    bundle install
+When running in a Docker container (option 2 above), the `asis` application is configured to run on port [3300](http://localhost:3300/). Required dependencies - ([Ruby](https://github.com/GSA/asis#ruby), and [Gems](https://github.com/GSA/asis#ruby)) - are installed using Docker. However, other data or configuration may need to be setup manually, which can be done in the running container using `bash`.
 
-### Services
+Using bash to perform any operations on asis application running in Docker container, below command needs to be run in `search-services`.
```

Comment:
Can we reword this?  Phrasing seems a little awkward and I'm not sure what it is referring to

Reply:
@MyNameIsMissing please review the updated phrase.